src/Stepper/Step.tsx

```diff
@@ -0,0 +1,111 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import "./Step.less";
+import CheckMark from "./CheckMark";
+import {FlexBox} from "../flex";
+import Exclamation from "./Exclamation";
+
+const propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.oneOfType([PropTypes.node]),
+  showTitle: PropTypes.bool,
+  showMessage: PropTypes.bool,
+  showDescription: PropTypes.bool,
```

Comment:
missed from last feedback round - let's drop the `show*` props and use the presence of the props to decide what's rendered. these just bloat the API with no added benefit